src/services/entities/CourtCase.ts

```diff
@@ -124,4 +124,11 @@ export default class CourtCase extends BaseEntity {
 
   @OneToMany(() => Note, (note) => note.courtCase)
   notes!: Relation<Note>[]
+
+  isLockedByAnotherUser(username: string) {
+    return (
+      (!!this.errorLockedById && this.errorLockedById !== username) ||
```

Comment:
I think I've come across this before, in the db we actually use the username to lock the record and not the user id right? Is this something we need/ want to carry on doing?

Reply:
or if not can `errorLockedById` -> `errorLockedByUsername`